### DISCUSSION TITLE
how to check if two lines intersect in haskell code ?

### DISCUSSION THREAD

**Discussion by flash24-cloud** [haskell/haskell-language-server#3341] [Q&A] [answered]
How to check if two lines intersect or not in Haskell ?

**Comment by fendor** [comment, marked as answer]
Hi, while it is great to see people learn Haskell, this is not the right location to ask such questions. Ask on [libera #haskell](https://libera.chat/), or on [r/learnhaskell reddit](https://www.reddit.com/r/learnhaskell) or various discord servers, e.g. [functional programming discord](https://discord.gg/SMJDSH6E)
